routes/api.php

```diff
@@ -310,6 +310,8 @@
                 'GoogleClassroomController@publishPlaylistToGoogleClassroom');
             Route::post('projects/{project}/playlists/{playlist}/activities/{activity}/publish',
                 'GoogleClassroomController@publishActivityToGoogleClassroom');
+            Route::post('activities/{activity}/publish',
```

Comment:
Can we use "{independent_activity}" instead of "{activity}" to stick to existing conventions for independent_activity param.